packages/tools/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -45,6 +45,7 @@ export default async args => {
     if (!["web", "native"].includes(platform)) {
         throw new Error("Must pass --configPlatform=web|native parameter");
     }
+    console.info(blue(`Project Path: ${projectPath}`));
```

Comment:
Better move this to the session where the actual projectPath is being used. Line 243

Reply:
Cool - Ive moved it